frontend/talentsearch/src/js/components/search/EstimatedCandidates.tsx

```diff
@@ -6,59 +6,74 @@ interface EstimatedCandidatesProps {
   updatePending?: boolean;
 }
 
-const EstimatedCandidates: React.FunctionComponent<
-  EstimatedCandidatesProps
-> = ({ candidateCount, updatePending }) => {
-  const intl = useIntl();
+const EstimatedCandidates: React.FunctionComponent<EstimatedCandidatesProps> =
+  ({ candidateCount, updatePending }) => {
+    const intl = useIntl();
 
-  function weight(msg: string) {
-    return updatePending ? (
-      <span className="lds-dual-ring" />
-    ) : (
-      <span data-h2-font-weight="b(800)">{msg}</span>
-    );
-  }
+    function weight(msg: string) {
+      return <span data-h2-font-weight="b(800)">{msg}</span>;
+    }
```

Comment:
Not sure if we have team conventions here, but for one-liner "components" I tend to omit the return:

```suggestion
    function weight(msg: string) (
      <span data-h2-font-weight="b(800)">{msg}</span>;
    )
```